Source/WebCore/page/Quirks.cpp

```diff
@@ -2469,6 +2478,11 @@ static void handleLiveQuirks(QuirksData& quirksData, const URL& quirksURL, const
     auto topDocumentHost = quirksURL.host();
     // outlook.live.com: rdar://136624720
     quirksData.needsMozillaFileTypeForDataTransferQuirk = topDocumentHost == "outlook.live.com"_s;
```

Comment:
Rather than do the string comparison three times (on iOS), maybe we could set the `quirksData.isOutlook` first (on all platforms), and then use that value to set `quirksData.needsMozillaFileTypeForDataTransferQuirk` (on all platforms), and `mayNeedToIgnoreContentObservation` on iOS?